src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] storage;
+    private int size;
+
+    public ArrayList() {
+        storage = new Object[DEFAULT_CAPACITY];
+    }
+
+    public ArrayList(int capacity) {
+        storage = new Object[capacity];
```

Comment:
what happens, if capacity will be negative number?
you should have 3 ways for this constructor:
- when capacity is more then 0 - create array with this capacity
- equal to 0 - create array with default size 
- less then 0 - throw exception with message